### PR TITLE
Fix http post actor delay.

### PR DESCRIPTION
**Pull request by vjkoskela**

### CODE REVIEW

**Review by paultgibbons — APPROVED** [review]
Thanks Ville!

**Review by paultgibbons — APPROVED** [review]

File: src/main/java/com/arpnetworking/tsdcore/sinks/HttpPostSinkActor.java [hunk 1]
```diff
@@ -227,22 +231,28 @@ private void processEmitAggregation(final EmitAggregation emitMessage) {
...
+                            WaitTimeExpired.getInstance(),
+                            context().dispatcher(),
+                            self());
+                } else if (!_waiting) {
+                   // If we have something in-flight continue to send without waiting
+                   dispatchPending();
+                }
+                // Otherwise we're already waiting, these requests will be sent after the waiting is over, no need to do anything else.
             } else {
+                // Spreading is disable, just keep dispatching the work
```
Comment (cwbriones) [inline src/main/java/com/arpnetworking/tsdcore/sinks/HttpPostSinkActor.java:255]: nit. typo disable**d**

File: src/main/java/com/arpnetworking/tsdcore/sinks/HttpPostSinkActor.java [hunk 2]
```diff
@@ -100,6 +100,10 @@ public HttpPostSinkActor(
         } else {
             _spreadingDelayMillis = new Random().nextInt((int) spreadPeriod.toMillis());
         }
+        LOGGER.info()
+                .setMessage("Http post sink actor spread period")
+                .addData("spreadPeriodMillis", _spreadingDelayMillis)
```
Comment (cwbriones) [inline src/main/java/com/arpnetworking/tsdcore/sinks/HttpPostSinkActor.java:105]: Can you some of the other configuration parameters here and change the message to "Starting HTTP post sink actor"

**Review by cwbriones — APPROVED** [review]
LGTM. you might be able to unnest this chain by turning the else(s) into early returns but it looks correct either way.

**Comment by cwbriones** [comment]
apparently my page was stale, I can see now that you already merged it 😅

**Pull request merged**